consensus/src/persistent_liveness_storage.rs

```diff
@@ -127,18 +139,29 @@ impl LedgerRecoveryData {
             .find(|qc| qc.certified_block().id() == root_block.id())
             .ok_or_else(|| format_err!("No QC found for root: {}", root_id))?
             .clone();
-        let root_ordered_cert = quorum_certs
-            .iter()
-            .find(|qc| qc.commit_info().id() == root_block.id())
-            .ok_or_else(|| format_err!("No LI found for root: {}", root_id))?
-            .clone();
 
+        let (root_ordered_cert, root_commit_cert) = if order_vote_enabled {
+            // We are setting ordered_root same as commit_root. As every committed block is also ordered, this is fine.
+            // As the block store inserts all the fetched blocks and quorum certs and execute the blocks, the block store
+            // updates highest_ordered_cert accordingly.
+            (
+                WrappedLedgerInfo::new(VoteData::dummy(), latest_ledger_info_sig.clone()),
+                WrappedLedgerInfo::new(VoteData::dummy(), latest_ledger_info_sig.clone()),
```

Comment:
nit: this duplicate is a bit ugly, we can just assign a single variable and clone it

Reply:
Done.